tests/test_utils_kernel.py

```diff
@@ -0,0 +1,51 @@
+from pyriemann.utils.kernel import kernel_riemann, kernel
+from pyriemann.utils.mean import mean_riemann
+from pyriemann.utils.base import logm
+
+from numpy.testing import assert_array_equal, assert_array_almost_equal
+import numpy as np
+
+from pyriemann.utils.test import is_sym
+
+
+def test_riemann_kernel(rndstate, get_covmats):
+    """Test Riemannian Kernel build"""
+    n_trials, n_channels = 5, 3
+    cov = get_covmats(n_trials, n_channels)
+    K = kernel_riemann(cov, cov, np.eye(n_channels))
+    assert is_sym(K)
```

Comment:
I think not, doesn't even have to be positive semi definite. 
I also stumbled across this, but I think that it's only a misconception from mercer's kernel theorem which says, that for each positive (semi) definite kernel there is a space where that kernel is the scalar product. 
But the other way around does't have to be true, it seems. Not every kernel matrix that comes from an arbitrary scalar product ist positive (semi) definite. Here, we are given the scalar product explicitly.

Reply:
Okay, there appear to be some numerical errors when computing the kernel matrix and its eigenvalues from the generated spds. There are some eigenvalues very close to 0, but in the negative domain. They should be 0, but aren't. There are only n_channels positive eigenvalues, the rest is (up to numerical errors) zero. Not quite sure how to handle this.

Reply:
So this is not a problem for the Embedding scenario, because it regularizes the kernel matrix anyways in the barycenter_weights function to make it positive definite as before it's only semi-definite. 
For other applications (like the RIemannian SVC that will be my next PR), this might be problematic. I'll have to think about this a bit more. Still unsure if we should implement it as it is or set the eigenvalues to zero somewhat manually.

Reply:
Three suggestions:
1. regularize kernel matrix in the kernel function by adding some scaled identity (I chose 1e-10) and remove regularization in riemann_lle.
2. compute eigenvalues, set eigenvalues with low absolute value (below 1e-10) to zero. Could still induce errors when recalculating the matrix with its eigenvectors and the adjusted eigenvalues.
3. Leave everything as it is and only test for symmetry.

I prefer 1.